### DISCUSSION TITLE
Settings -> Field Type "Entries" -> Selectable Entries Condition -> condition type "Descendant of"

### DISCUSSION THREAD

**Discussion by mfell** [craftcms/cms#15518] [Ideas]
It's not possible, to create a condition to get all descendants of certain parent id(s).  
I think, it would be very useful to have this selection criteria within the Craft core?

My idea is regarding to this discussion on stackexchange:  
https://craftcms.stackexchange.com/questions/41892/field-selectable-entries-condition-get-all-descendants-of-certain-id

My solution in the screenshot is NOT working, but it should clarify, what I'm missing.

### Section type "structure"
![9QTtR6fK](https://github.com/user-attachments/assets/f1d93286-4205-4d57-96bd-c3627231c0b2)

### Settings, field definition (not working, of course)
![H3Z0XbbO](https://github.com/user-attachments/assets/795dff55-9676-40f7-8e7d-8aed90fde3fc)

**Comment by sweetroll** [comment]
This is something we have been needing in recent projects too. Having descendant/ancestor options would be great or even a field that resolves to an array of selectable IDs would allow the above query to work as well as many other uses cases.

**Comment by sweetroll** [comment]
@mfell did you find a solution for this?

Having selectable entries dictated by the state of the owner entry is something that we are increasingly in need of. As mentioned above, even a 'has ID' field would be a great start.

**Reply by mfell** [reply]
Hmm ... as far as I remember, I followed the hint within the link on stackexchange (see above).
